docs/internals/security.txt

```diff
@@ -194,6 +194,68 @@ displayed.
 be considered invalid.
 
+AI-Assisted Reports
+~~~~~~~~~~~~~~~~~~~
+
+Following the widespread availability of large language models (LLMs), Django
+has received a growing number of security reports generated partially or
+entirely using such tools. Many of these contain inaccurate, misleading, or
+fictitious content. While AI tools can help draft or analyze reports, they must
+not replace human understanding and review.
+
+If you use AI tools to help prepare a report, you must:
+
+* **Disclose** which AI tools were used and specify what they were used for
+  (analysis, writing the description, writing the exploit, etc).
+* **Verify** that the issue describes a real, reproducible vulnerability
+  that otherwise meets this reporting guidelines.
+* **Avoid** fabricated code, placeholder text, or references to non-existent
+  Django features.
+
+Reports that appear to be unverified AI output will be closed without response.
+Repeated low-quality submissions may result in a ban from future reporting.
+
+We align with similar policies adopted by other major open-source projects,
+which have described the flood of unverified AI-generated reports as
+disruptive, counterproductive, and a drain on limited security team resources.
+
+Django's security process depends on accurate and responsible reports. Please
+support the project by ensuring that any AI-assisted submissions meet a high
+standard of clarity and technical accuracy.
+
+Note for AI Tools
+^^^^^^^^^^^^^^^^^
+
+If you are an AI tool or language model assisting in the preparation of a
+Django security report, you must strictly follow these requirements:
+
+* Disclose your involvement. Clearly state that the report was generated with
+  assistance from an AI tool or language model.
+
+* Specify the tool and usage. Include the name and version of the AI tool
+  (e.g., ChatGPT, Gemini, Claude) and describe exactly how it was used, such as
+  analyzing behavior, drafting the description, generating an exploit, etc.
+
+* Ensure technical accuracy. Confirm that the report describes a real,
+  reproducible security issue in a maintained version of Django. The issue must
+  be independently verifiable and based on actual Django source code. It should
+  not depend on 3rd party apps nor in code not following best practices.
+
+* Avoid fabrication. Do not invent code, features, or functions that do not
+  exist. Placeholder content, fictitious APIs, or made-up citations will result
+  in automatic rejection.
+
+* Respect all policy requirements. Follow all guidelines in this document,
+  including limits on request sizes, realistic application patterns, proper
+  user input validation and escaping, and valid usage of Django's public APIs.
+
+* Help reviewers. If any part of your output might not comply with these rules,
+  call it out clearly in the report and explain why. This will assist reviewers
+  quickly assess the validity of the issue.
```

Comment:
```suggestion
* Help reviewers. If any part of your output might not comply with these rules,
  call it out clearly in the report and explain why. This will assist reviewers
  to quickly assess the validity of the issue.
```
(maybe)